### DISCUSSION TITLE
Party mode instant music button

### DISCUSSION THREAD

**Discussion by Ecto11** [gpstar81/GPStar-proton-pack#417] [Ideas]
Could you add a feature where you press the barrel switch when in "party mode" and it plays a specific WAV song in your playlist? or using that as a faster way to start playing the music? {instead of going through the menu}

**Comment by nomakewan** [comment]
I don't think we have a "party mode" in our software. Could you clarify what you mean here?

**Comment by Ecto11** [comment]
This would be when the cyclotron and light bar rotates through the RGB colours instead of the standard red and blue colours.. the wand barrel button currently does nothing on this mode.. hoping to use that to play a particular sound file entering a "party mode"

Thanks,

Dave McCauley
***@***.***
0433820583
________________________________

**Reply by nomakewan** [reply]
Ah, the Spectral firing mode. No, I don't think we'll be tying any music into it.

It should be fairly fast to get to music though even if you were in that firing mode already. Flip down top-right switch, hit barrel wing button, flick top knob down to option 1, hit intensify. Don't even have to return to the top of the menu after either; flicking the top switch back up will automatically exit the menu and put you back into firing readiness.

To elaborate further, merely putting a "play" function on the barrel wing button in firing mode is a non-starter request. There would need to be other playback controls available--which there are in the menu already, and said menu is easy to access. Additionally if you install an ESP32 in your pack you could either use your phone to control music playback at any time regardless, or could even create your own wireless device that uses the web API to perform the same tasks with a button press or similar.
